### ISSUE TITLE
copying into memory does not work

### ISSUE THREAD

**Issue by notepaddpp1023** [notepad-plus-plus/notepad-plus-plus#14357]
The issue was present and still is in the latest version.

1. In the software with Find and Replace window open, when switching with Alt_Tab in windows, CTRL+C does not copy to memory then in the find window. To successfully copy, multiple CTRL+C need to be pressed. When working fast, this issue results in replacing with a different string since CTRL+C does not work the first time.
2. in firefox, cannot create account for community support. Alternative methods to login are not displayed in the login page. other browsers are not available. Please provide the links to these methods so that I can post in the community. No comments on the elected method of community participation. google remains the big brother but even people who think they are smart do not see this. Maybe they see it but have their pockets lined up. whatev

**Comment by alankilborn** [comment]
The issue template (that you didn't fill out) helps developers recreate your problem (see _Steps to Reproduce_ section), because it guides you to being very specific.  Reading what you _did_ write, I can't really tell how I'd begin to try your problem.

"Current version" is also meaningless.  It may be a long time before someone that could fix it examines your problem.  The current version then may be different from the current version as of now.

In short, filling out the template instead of ignoring it would solve all of these problems...

**Comment by donho** [comment]
> In short, filling out the template instead of ignoring it would solve all of these problems...

👍 

Issue closed for the incomplete info.
For any user who creates issue: if you want to get help, provide all the information so people can help you.
